src/scripts/npm/updateNativeSdk.sh

```diff
@@ -128,7 +116,7 @@ main() {
   fi
 
   # update plugin.xml
-  gulp prod
+  #gulp prod
```

Comment:
Do you mean to have this commented out?

Reply:
Yes.  The gulpfile was removed with version 2.7.0, so this line errors out.  It appears it was used to update the xml file with all the iOS filenames.